debug/test/browser/debug.test.js

```diff
@@ -383,6 +383,109 @@ describe('debug', () => {
 		});
 	});
 
+	describe('table markup', () => {
+		it('missing <tbody>/<thead>/<tfoot>', () => {
+			const Table = () => (
+				<table>
+					<td>Hi</td>
+				</table>
+			);
+			render(<Table />, scratch);
+			expect(console.error).to.be.calledOnce;
+		});
+
+		it('missing <tr>', () => {
+			const Table = () => (
+				<table>
+					<tbody>
+						<td>Hi</td>
+					</tbody>
+				</table>
+			);
+			render(<Table />, scratch);
+			expect(console.error).to.be.calledOnce;
+		});
+
+		// Don't know how to do this yet.
+		it.skip('missing <td>', () => {
+			const Table = () => (
+				<table>
+					<tbody>
+						<tr>
+							<div />
+						</tr>
+					</tbody>
+				</table>
+			);
+			render(<Table />, scratch);
+			expect(console.error).to.be.calledOnce;
+		});
+
+		it('Using <td> instead of <th> in <thead>', () => {
+			const Table = () => (
+				<table>
+					<thead>
+						<tr>
+							<td>Hi</td>
+						</tr>
+					</thead>
+				</table>
```

Comment:
That's a valid table per spec. It passes the validator on https://validator.w3.org/nu/ . Relevant excerpts from the spec: [tr](https://html.spec.whatwg.org/multipage/tables.html#the-tr-element), [thead](https://html.spec.whatwg.org/multipage/tables.html#the-thead-element).

`thead` content model:

> Zero or more `tr` and script-supporting elements.

`tr` content model:

> Zero or more `td`, `th`, and script-supporting elements.